src/components/SquareBox.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react'
+//creates SquareBox class
+class SquareBox extends Component{
+    constructor(props){
+      super(props)
+      this.state = {
+        color: ["white", "yellow"],  // array of two colors
+        lightStatus: ["OFF", "ON"],  // array of two status
+        index: 0 // index is the position of the array, in this case: 0 ===> "OFF" ===> "white" (by default)
+      }
+    }
+    // status setter
+    changeStatus = () => {
+        //this.state.index will always change between 0 and 1, since index modular by 2, the result will always be 0 or 1, aka "OFF" and "ON"
+        this.setState({index: (this.state.index + 1) % 2 })
```

Comment:
that's pretty clever, i like this solution!

Reply:
Thanks bro, I am so happy it works.